packages/splitter/src/useSplitter.ts

```diff
@@ -288,15 +345,24 @@ export function useSplitter({
     onTouchStart,
     'aria-label': ariaLabel,
     'aria-controls': primaryPaneId,
-    'aria-valuenow': dimension,
+    'aria-valuenow': separatorPosition,
     'aria-valuemin': min,
     'aria-valuemax': max,
     'aria-orientation': orientation,
     tabIndex: 0,
     ...other
   });
 
+  const getPrimaryPaneProps = ({ ...other } = {}) => ({
+    'data-garden-container-id': 'containers.splitter.primaryPane',
+    'data-garden-container-version': PACKAGE_VERSION,
+    id: primaryPaneId,
+    valueNow: separatorPosition,
+    ...other
+  });
```

Comment:
Manage the `id` from `useId` on the primary pane as well as deliver `valueNow` directly to the primary pane.

Reply:
Fixed in https://github.com/zendeskgarden/react-containers/pull/405/commits/f2ee30a39269af3af40b9dc1c59a6b20de5edd03